src/views/Styleguide.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <div></div>
+</template>
+
+<script>
+export default {
+  name: "Styleguide"
```

Comment:
Always use single quotes for imports, components, properties, etc 
name: 'Styleguide'